<launch>

	<node pkg="localization_node" type="localization_node_node" name="map" />

</launch>
